st2actions/st2actions/query/mistral/v2.py

```diff
@@ -115,9 +113,13 @@ def _get_workflow_result(self, exec_id):
         """
         execution = executions.ExecutionManager(self._client).get(exec_id)
 
-        output = jsonify.try_loads(execution.output) if execution.state in DONE_STATES else None
+        result = jsonify.try_loads(execution.output) if execution.state in DONE_STATES else {}
+        result['workflow'] = {
+            'state': execution.state,
```

Comment:
Yes, sometimes when Mistral fails at the WF level (not task level), it'll put additional state information in state_info.  We should pull that info over for the user so when the WF fails, the info is available.